lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
If this is calling `db.insert()` behind the scenes then there is no reason to perform an additional `db.get()`. Simply add the returned `_rev` to the local copy of the document and use that. CouchDB/Cloudant does not support partial updates, only whole-document updates, so any operation to update a document involves constructing the full document locally and then pushing it to the DB.